src/test/java/VisibilityAndUrlCorrectnessTest.java

```diff
@@ -0,0 +1,50 @@
+import desktop.fragments.NavigationBar;
+import desktop.pages.SignInPage;
+import driver.SingletonDriver;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+
+import org.assertj.core.api.SoftAssertions;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static constants.Constants.*;
+import static org.junit.Assert.assertEquals;
+
+public class VisibilityAndUrlCorrectnessTest {
+    private final WebDriver driver = SingletonDriver.getDriver();
+    private final List<String> params = new ArrayList<String>() {{
```

Comment:
if you use Java 11+ - you can use a shorten version:
`private final List<String> params = List.of(NAME, EMAIL, PASSWORD);`

for Java 8 it can be like this:
`private final List<String> params = Arrays.asList(NAME, EMAIL, PASSWORD);`